src/modules/overview/components/PromoSkeleton/PromoSkeleton.tsx

```diff
@@ -0,0 +1,126 @@
+import {Box, Container, Grid, Hidden, Typography} from '@material-ui/core';
```

Comment:
```suggestion
import {Box, Container, Grid} from '@material-ui/core';
```
I suggest to remove it as it is not being used.